### PR TITLE
#156 Upgrade node version to 12.18.2

### PR DESCRIPTION
**Pull request by aqtang**
Addresses https://github.com/elyra-ai/canvas/issues/156

Upgrade node version in travis
 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,15 +20,15 @@ branches:
...
+node_js: "12.18.2"
 addons:
   apt:
     packages:
       - google-chrome-stable
       - unzip
 before_install:
   - . ./scripts/setEnv.sh
-  - npm install -g npm@5.6.0
+  - npm install -g npm@6.14.5
```
Comment (matthoward366) [inline .travis.yml:31]: We should just remove this and default to the node_js version.

File: .travis.yml [hunk 2]
```diff
@@ -20,15 +20,15 @@ branches:
     - master-last_success
     - release-*
 language: node_js
-node_js: "12"
+node_js: "12.18.2"
```
Comment (matthoward366) [inline .travis.yml:23]: We should keep this generic.  Does it use the latest if we have 12 or do we need 12.x?

Comment (aqtang) [reply .travis.yml:23]: `12 latest 12.x release` 
I'll revert that, it will be generic enough 
12 latest 12.x release

File: .travis.yml (hunk 1 above)
Comment (aqtang) [reply .travis.yml]: removed

**Pull request merged**